src/main/resources/config/liquibase/changelog/20190929121500_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet id="20190929121500" author="phuong.anh.montag">
+        <addColumn tableName="course">
+            <column name="presentation_score" defaultValueNumeric="0" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <addColumn tableName="exercise">
+            <column name="presentation_score_enabled" defaultValueNumeric="0" type="bit">
```

Comment:
As I saw [here](https://www.liquibase.org/documentation/column.html) you can also specify `BOOLEAN` and it will be correctly handled by Liquibase. I would suggest to do this, since it is more clear. Also you can use `defaultValueBoolean` to set default value

Reply:
Thank you for the suggestion, I changed that :)